app/models/vote.rb

```diff
@@ -0,0 +1,11 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates :vote_value,
+            inclusion: { in: [1, -1] },
+            presence: true#,
+            # uniqueness: {scope: :review, message: ": You can't upvote twice"}
+  validates :user, presence: true
+  validates :review, presence: true
```

Comment:
When we tried this, it seemed to not allow *any* duplicate votes for a given article - so there could be one up, and one down, but even if someone else tried to vote, it wouldn't work. That's what seemed like it happened, anyway...